server/Dockerfile

```
@@ -1,16 +1,23 @@
-FROM python:3.11
+FROM python:3.11-slim
```

`slim` causes errors in macos (gcc apparently isn't included in the image)
```suggestion
FROM python:3.11
```